Task.rb

```diff
@@ -0,0 +1,170 @@
+require 'watir'
```

Comment:
We recommend you to read the Ruby Style Guide and update your code.
https://rubystyle.guide